judo-ui-react/src/main/resources/actor/src/utilities/form-utils.ts.hbs

```diff
@@ -1,5 +1,10 @@
-import type { Dayjs } from 'dayjs';
+export const uiDateToServiceDate = (date?: any | null): string | null => {
+  return date;
+};
 
-export const uiDateToServiceDate = (date?: Dayjs | null): string | null => {
-  return date ? date.format().substring(0, 10) : null;
+export const serviceDateToUiDate = (dateStr?: any) => {
+  if (typeof dateStr === 'string') {
+    return dateStr.endsWith('Z') ? dateStr : (dateStr + 'Z');
+  }
+  return dateStr;
```

Comment:
Have to extend the endWidth, because it can `+[0[0[:00]]` form also valid as ISO 8601 is defining. On that type of date extra invalid Z will be added.

Reply:
This is a temporary workaround for the current backend issue. Will be removed once the backend is fixed.